### ISSUE TITLE
[BUG] As of Version 2.14.0 it is not possible to add Actions to Monitors in the Alerting section.

### ISSUE THREAD

**Issue by proligde** [opensearch-project/OpenSearch-Dashboards#7004]
**Describe the bug**

As of Version 2.14.0 it is not possible to add Actions to Monitors in the Alerting section.

**To Reproduce**
1. Make sure there are existing Channels that Alerting Actions could use
2. Create a Monitor that contains a trigger in the Alerting Section of Opensearch Dashboards. Try adding an action to that trigger

**Expected behavior**
I can configure the Action

**Actual behaviour
The Actions Section states there would be no Channels to use. I can only click on the link to the Channels Management, where my Channels are shown without problem.

**OpenSearch Version**
2.14.0

**Dashboards Version**
Problem exists in 2.14.0 only. Rolling back to 2.13.0 (while keeping opensearch 2.14.0) solves the issue

**Plugins**

opensearch-alerting
opensearch-anomaly-detection
opensearch-asynchronous-search
opensearch-cross-cluster-replication
opensearch-custom-codecs
opensearch-flow-framework
opensearch-geospatial
opensearch-index-management
opensearch-job-scheduler
opensearch-knn
opensearch-ml
opensearch-neural-search
opensearch-notifications
opensearch-notifications-core
opensearch-observability
opensearch-performance-analyzer
opensearch-reports-scheduler
opensearch-security
opensearch-security-analytics
opensearch-skills
opensearch-sql

**Screenshots**

Screenshot of the JS console in 2.14.0 (doesn't work)

![opendashboards-error](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/998665/9d719367-c220-4c98-88e2-f6786edeca60)

Opening the target of the 400-error request I ged:

```
{"statusCode":400,"error":"Bad Request","message":"[request query.config_type]: expected at least one defined value but got [undefined]"}
```

Screenshot of the JS console in 2.13.0 (works)

![opendashboards-working](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/998665/2356e3d6-0eea-456e-8b77-e69ca680900b)

**Host/Environment (please complete the following information):**
 - Ubuntu 22.04 LTS (Host-System)
 - I'm using the official Helm-Chart on Kubernetes (v. 2.18.0 and 2.17.0 for downgrading)
 - Tested with current Chrome and Firefox

**Comment by Hailong-am** [comment]
same as https://github.com/opensearch-project/alerting-dashboards-plugin/issues/961

**Comment by proligde** [comment]
@Hailong-am thank you!- I was searching for that issues before posting but didn't find any. Since it seems to be fixed in upcoming 2.15 I close this ticket.
